### PR TITLE
Updated config.yaml to use 2.11 branch reference

### PR DESCRIPTION
**Pull request by dislbenn**
# Description

Updated `bundle-automation/config.yaml` to use current release branch ref.

## Related Issue

If applicable, please reference the issue(s) that this pull request addresses.

## Changes Made

Updated the `bundle-automation/config.yaml` to use `release-2.11` branch ref.

## Screenshots (if applicable)

Add screenshots or GIFs that demonstrate the changes visually, if relevant.

## Checklist

- [x] I have tested the changes locally and they are functioning as expected.
- [ ] I have updated the documentation (if necessary) to reflect the changes.
- [ ] I have added/updated relevant unit tests (if applicable).
- [x] I have ensured that my code follows the project's coding standards.
- [x] I have checked for any potential security issues and addressed them.
- [x] I have added necessary comments to the code, especially in complex or unclear sections.
- [x] I have rebased my branch on top of the latest main/master branch.

## Additional Notes

Add any additional notes, context, or information that might be helpful for reviewers.

## Reviewers

Tag the appropriate reviewers who should review this pull request. To add reviewers, please add the following line: `/cc @reviewer1 @reviewer2`

/cc @cameronmwall @ngraham20 

## Definition of Done

- [x] Code is reviewed.
- [x] Code is tested.
- [x] Documentation is updated.
- [x] All checks and tests pass.
- [x] Approved by at least one reviewer.
- [x] Merged into the main/master branch.

### CODE REVIEW

File: hack/bundle-automation/config.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 - repo_name: multicloud-operators-subscription
   github_ref: "https://github.com/stolostron/multicloud-operators-subscription.git"
-  branch: "release-2.10"
+  branch: "main"
```
Comment (dislbenn) [inline hack/bundle-automation/config.yaml:4]: Using `main` branch as reference until `release-2.11` branch is created for `multicloud-operators-subscription`.

File: hack/bundle-automation/copy-config.yaml [hunk 2]
```diff
@@ -4,6 +4,6 @@
     - name: "cluster-permission"
       chart-path: "chart/"
       always-or-toggle: "toggle"
-      branch: "release-2.10"
+      branch: "main"
```
Comment (dislbenn) [inline hack/bundle-automation/copy-config.yaml:7]: Using `main` branch as reference until `release-2.11` branch is created for `cluster-permission`.

File: hack/bundle-automation/config.yaml (hunk 1 above)
Comment (dislbenn) [reply hack/bundle-automation/config.yaml]: Verified that the `release-2.11` was created. Updated the config file to reflect those changes.

File: hack/bundle-automation/copy-config.yaml (hunk 2 above)
Comment (dislbenn) [reply hack/bundle-automation/copy-config.yaml]: Verified that the `release-2.11` was created. Updated the config file to reflect those changes.

**Review by ngraham20 — APPROVED** [review]
/lgtm

**Review by ngraham20 — APPROVED** [review]
/lgtm

**Pull request merged**